src/components/StackedAvatar.tsx

```diff
@@ -8,7 +8,7 @@ const StackedAvatar = ({ contributors }: StackedAvatarProps) => (
     {contributors && contributors.slice(0, 5).map(({ contributor, last_merged_at }) => (
```

Comment:
```suggestion
    {contributors?.slice(0, 5).map(({ contributor, last_merged_at }) => (
```
This is complaining, but relevant. Can we test this locally?

Reply:
yea sure its working perfectly before i pushed

Reply:
Yea sure working perfectly